app/controllers/provider_foreman_controller.rb

```diff
@@ -36,12 +36,12 @@ def show_list
   def new
     assert_privileges("provider_foreman_add_provider")
     @provider_cfgmgmt = ManageIQ::Providers::ConfigurationManager.new
-    @provider_types = ["Ansible Tower", "Foreman"]
+    @provider_types = ["Ansible Tower", ui_lookup(:ui_title => 'foreman')]
```

Comment:
This is just bad in general because it doesn't honor the permissions.yml.  Everything should really go through the methods like `ExtManagementSystem.supported_types_and_descriptions_hash` cc @dclarizio